docs/IWYUStyle.md

```diff
@@ -281,6 +281,23 @@ comment:
 The args are added to the `include-what-you-use` command as-written after line
 unwrapping, so IWYU flags must be explicitly prefixed with `-Xiwyu`.
 
+### Prerequisites ###
+
+If a test has prerequisites, annotate the `.cc` file itself using a
+directive comment:
+
+```
+// IWYU_REQUIRES: feature(value)
+// IWYU_UNSUPPORTED: feature(value)
+```
+
+`IWYU_REQUIRES` states that the following feature test must evaluate
+true for the test to be valid.
+
+`IWYU_UNSUPPORTED` states that the test is invalid when the feature
+test evaluates as false.
```

Comment:
I got this line wrong :(, the test is invalid when this feature test is true. Flipped too many things at once

Reply:
Good catch, fixed!